syncer/peers/nofork/proto/syncer.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package v1;
+
+option go_package = "/syncer/peers/nofork/proto";
+
+import "google/protobuf/empty.proto";
+
+service SyncPeer {
+  rpc GetBlocks(GetBlocksRequest) returns (stream Block);
+  rpc GetBlock(GetBlockRequest) returns (Block);
+  rpc GetStatus(google.protobuf.Empty) returns (Status);
+  rpc NotifyStatus(Status) returns (google.protobuf.Empty);
```

Comment:
Why are you diverging from the spec we've agreed upon?

In the spec we've specifically decided to eliminate this status fetching and explicit notify RPC calls.
Why did you reintroduce them back, as they add an insane network and flow overhead?

Reply:
I confirmed about Gossip with this morning and I've not worked on this yet.
I will let you know once I pushed this change

Reply:
@zivkovicmilos 
Removed 17fd0aa3cb63d501b009ab85867dee90fe26ae8d